examples/blog-starter/package.json

```diff
@@ -1,24 +1,24 @@
 {
   "private": true,
   "scripts": {
-    "dev": "next",
+    "dev": "next dev --turbopack",
     "build": "next build",
     "start": "next start"
   },
   "dependencies": {
     "classnames": "^2.5.1",
     "date-fns": "^3.6.0",
     "gray-matter": "^4.0.3",
-    "next": "latest",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
+    "next": "15.0.2",
```

Comment:
We should always use `latest` in example so that when they do create-app from github example, they can get latest nextjs installed, and we don't manually update them later.